client/src/entities/player.rs

```diff
@@ -64,6 +64,7 @@ where F: Fn () -> B
         spread: 2.0,
         shot: weapon::Shot::Single,
         bullet_speed: 200.0,
+        pellet_number: 1,
```

Comment:
What is the meaning of `pellet_number`? Isn't `1` redundant to `Shot::Single`?

Reply:
No. 'Shot' means that how many firing happens one after one on one trigger pull. Pellet (hun: sörét) is the number of particles being shot per firing.
Maybe all the guns which shoot multiple pellets will be single-shot weapons but it is an orthogonal dimension of the weapondetails